modules/planning/tasks/deciders/open_space_decider/open_space_roi_decider.cc

```diff
@@ -298,20 +298,21 @@ void OpenSpaceRoiDecider::SetPullOverSpotEndPose(Frame *const frame) {
 }
 
 void OpenSpaceRoiDecider::SetParkAndGoEndPose(Frame *const frame) {
-  const double kSTargetBuffer = 5.0;
-  const double kSpeedRatio = 0.8;  // after adjust speed is 80% of speed limit
+  const double kSTargetBuffer = 8.0;
+  const double kSpeedRatio = 0.3;  // after adjust speed is 80% of speed limit
   // get vehicle current location
   const ReferenceLineInfo &reference_line_info =
       frame->reference_line_info().front();
 
   ADEBUG << "reference_line ID: " << reference_line_info.Lanes().Id();
   const auto &reference_line = reference_line_info.reference_line();
   // get vehicle s,l info
-  const auto &park_and_go_status =
-      PlanningContext::Instance()->planning_status().park_and_go();
+  auto park_and_go_status = PlanningContext::Instance()
```

Comment:
Why use mutable if you intend to just get `adc_init_x` and `adc_init_y` from `PlanningContext`? Or am I mis- understanding here?

Reply:
Because I need to write something into PlanningContext later.